src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenConstructorInitTests.cs

```diff
@@ -367,7 +367,7 @@ .maxstack  2
   IL_0003:  stfld      ""int S.<X>k__BackingField""
   IL_0008:  ldarg.0
   IL_0009:  ldarg.0
-  IL_000a:  call       ""int S.X.get""
+  IL_000a:  call       ""readonly int S.X.get""
```

Comment:
One quirk of this work: after a property is emitted we have no decent way of knowing whether it was implicitly readonly in source. So when the member is in a non-readonly type we just display `readonly` if the IsReadOnlyAttribute is present on the member.

I tried to avoid displaying `readonly` where it was not present in source, and for members of non-readonly types, because it minimizes test churn and limits showing the keyword to cases where its presence or absence will be more informative.